zeg/http.py

```diff
@@ -83,10 +83,11 @@ def make_session(endpoint, token):
 def handle_response(response):
     is_204 = response.status_code == 204
     is_200 = response.status_code == 200
+    is_201 = response.status_code == 201
```

Comment:
Should probably just be `is_success =  200 <= response.status_code < 300` at this point.